sim/dut-simulator.wake

```diff
@@ -90,47 +90,53 @@ tuple DUTSimCompileOptions =
   global Defines:        List NamedArg
   global SourceFiles:    List Path
   global Plusargs:       List NamedArg
+  global Resources:      List String
```

Comment:
Might be worth adding a comment here that describes the fact that this should be the same concept of "Resources" as in a Wake `Plan`.